client/containers/RemoveAgendaConfirmation.jsx

```diff
@@ -29,7 +29,10 @@ RemoveAgendaConfirmationComponent.propTypes = {
 const mapStateToProps = (state, ownProps) => ({
     plannings: Object.keys(selectors.getStoredPlannings(state))
         .filter((pKey) => (get(ownProps.agenda, 'planning_items', []).indexOf(pKey) > -1))
-        .map((pKey) => (selectors.getStoredPlannings(state)[pKey])),
+        .map((pKey) => {
+            return extend({}, selectors.getStoredPlannings(state)[pKey],
```

Comment:
Why not using the spread operator instead ? 
`{ ...selectors.getStoredPlannings(state)[pKey], _agenda: { name: ownProps.agenda.name }}`